components/Slider/src/MDCSlider.m

```diff
@@ -209,6 +213,25 @@ - (UIColor *)backgroundTrackTickColorForState:(UIControlState)state {
   return color;
 }
 
+- (void)setThumbRadius:(CGFloat)thumbRadius forState:(UIControlState)state {
+  NSAssert(thumbRadius >= 0, @"Thumb radius must be a positive value");
```

Comment:
Please add code to handle this case in non-debug code.

Reply:
Going to use the absolute value.